backend/apps/innogram/src/chat/pipes/ws-validation.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, ValidationPipe } from '@nestjs/common';
+import { WsException } from '@nestjs/websockets';
+import { ValidationError } from 'class-validator';
+import { CHAT_ERROR_MESSAGES } from 'libs/shared/src/constants/chat.constants';
```

Comment:
```suggestion
import { CHAT_ERROR_MESSAGES } from 'libs/shared/src/constants/chat.constants';
const ChatErrorMessages = {
  INVALID_MESSAGE: "message...",
  INVALID_MESSAGE2: "message...2",
}
```
better make object with constants like this because in it looks like CONSTANT.CONSTANT
also chat.constants.ts in inside of constants directiory so name of this should be chat.constant.ts not constants, because when we have file with interfaces we name it like interfaces/user.interace.ts no user.interfaces.ts

Reply:
done